lib/miq_automation_engine/service_models/miq_ae_service_container_image.rb

```diff
@@ -13,6 +13,7 @@ class MiqAeServiceContainerImage < MiqAeServiceModelBase
     expose :openscap_rule_results,        :association => true
     expose :exposed_ports,                :association => true
     expose :environment_variables,        :association => true
+    expose :image_ref
```

Comment:
Thanks @gmcculloug I was not aware of that.

@aweiteka can you confirm you have what you need (since 5.7.2 if I remember correctly)?